### PR TITLE
Injecting reagent containers sends nearby mobs a warning

### PR DESCRIPTION
**Pull request by 1Sparky1**
Title.

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Use visible_message instead of a manual loop.

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -261,7 +259,7 @@
 				admin_inject_log(user, target, src, contained, reagents.get_temperature(), trans)
 			else
 				trans = reagents.trans_to(target, amount_per_transfer_from_this)
-			to_chat(user, SPAN_NOTICE("You inject [trans] units of the solution. The syringe now contains [src.reagents.total_volume] units."))
+			user.visible_message(SPAN_WARNING("[user] injects [target] with the syringe!"),SPAN_NOTICE("You inject [trans] units of the solution. The syringe now contains [src.reagents.total_volume] units."),2)
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/syringes.dm:262]: Our reagent container might not necessarily be named a syringe. Use `with [src]` instead.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/syringes.dm (hunk 1 above)
Comment (1Sparky1) [inline code/modules/reagents/reagent_containers/syringes.dm:262]:
```suggestion
			user.visible_message(SPAN_WARNING("[user] injects [target] with \the [src]!"),SPAN_NOTICE("You inject [trans] units of the solution. \The [src] now contains [src.reagents.total_volume] units."),2)
```

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Casper3667 — APPROVED** [review]

**Pull request merged**